lib/reference/properties.mjs

```diff
@@ -644,3 +644,146 @@ export const longhandTimeProperties = new Set([
 ]);
 
 export const shorthandTimeProperties = new Set(['transition', 'animation']);
+
+export const pageContextProperties = new Set([
+	'direction',
+	'background-color',
+	'background-image',
+	'background-repeat',
+	'background-attachment',
+	'background-position',
+	'background',
+	'border-top-width',
+	'border-right-width',
+	'border-bottom-width',
+	'border-left-width',
+	'border-width',
+	'border-top-color',
+	'border-right-color',
+	'border-bottom-color',
+	'border-left-color',
+	'border-color',
+	'border-top-style',
+	'border-right-style',
+	'border-bottom-style',
+	'border-left-style',
+	'border-style',
+	'border-top',
+	'border-right',
+	'border-bottom',
+	'border-left',
+	'border',
+	'counter-reset',
+	'counter-increment',
+	'color',
+	'font-family',
+	'font-size',
+	'font-style',
+	'font-variant',
+	'font-weight',
+	'font',
+	'height',
+	'min-height',
+	'max-height',
+	'line-height',
+	'margin-top',
+	'margin-right',
+	'margin-bottom',
+	'margin-left',
+	'margin',
+	'outline-width',
+	'outline-style',
+	'outline-color',
+	'outline',
+	'padding-top',
```

Comment:
question
minor

Are `orphans` and `widows` missing from the list?
I am not really sure.

Here are some links that seems to confirm it:

https://olevelexam.com/css-property-orphans
https://web.archive.org/web/20150510080955/https://developer.mozilla.org/en-US/docs/Web/CSS/@page
https://publishing-project.rivendellweb.net/css-paged-media/#controlling-widows-and-orphans

I couldn't find any examples for `page-break-before`/`page-break-after`/`page-break-inside` though.

@romainmenke you might know more?
Are these examples invalid?

Reply:
Let's start with the ones from the spec in this PR. If a user reports `orphans` and `widows` we can handle those in another PR.